File: blinkreceipt-demo/BlinkReceiptDirectApi/src/main/java/com/blinkreceipt/ocr/BlinkApplication.java
package com.blinkreceipt.ocr;

import android.app.Application;

import com.microblink.BlinkReceiptSdk;

public class BlinkApplication extends Application {

    @Override
    public void onTerminate() {
        BlinkReceiptSdk.terminate();

        super.onTerminate();
    }

}

File: blinkreceipt-demo/BlinkReceiptOutlook/src/main/java/com/blinkreceipt/digital/outlook/BlinkApplication.kt
package com.blinkreceipt.digital.outlook

import android.app.Application
import android.util.Log
import com.microblink.core.InitializeCallback
import com.microblink.digital.BlinkReceiptDigitalSdk

class BlinkApplication : Application() {

    override fun onCreate() {
        super.onCreate()

        BlinkReceiptDigitalSdk.initialize(this, object : InitializeCallback {

            override fun onComplete() {
                Log.d(TAG, "initialize complete")
            }

            override fun onException(e: Throwable) {
                Log.d(TAG, "failed in initialize", e )
            }

        })
    }


    private companion object {
        const val TAG = "ImapApplication"
    }
}